README.md

```diff
@@ -159,6 +159,7 @@ Unless otherwise noted these projects are open-source with permissive licensing:
 - [nat/openplayground](https://github.com/nat/openplayground)
 - [Faraday](https://faraday.dev/) (proprietary)
 - [LMStudio](https://lmstudio.ai/) (proprietary)
+- [Layla] (https://play.google.com/store/apps/details?id=com.laylalite) (proprietary)
```

Comment:
There's an extra space between `[Layla]` and the link which prevents this from rendering correctly

Reply:
Fixed!